### PR TITLE
Added Mood View and controller

### PR DESCRIPTION
**Pull request by mzc5387**
Added Mood view and controller, and read/write moods to and from a csv file.

### CODE REVIEW

File: src/foodmood/MoodController.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+     */
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        // TODO
+    }
+    
+    // Add a mood to the 'database'
+    public void addMood(){
+        Mood m = new Mood(1,"Happy");
+        Mood m1 = new Mood(1,"Sad");
```
Comment (redjen) [inline src/foodmood/MoodController.java:43]: This should have a different ID than the other one.

**Comment by redjen** [comment]
I think this should be OK, but Javs throws an exception when trying to load it (if you change the file name in TestLauncher you can see this) and doesn't log anything useful. It should work, though. I'll keep looking at it later. But I think this should be fine to merge in with the little changes I noted.

**Comment by redjen** [comment]
The other change is that you'll need to call your methlds in initialize. I can't comment on individual lines on mobile, it seems.

**Comment by mzc5387** [comment]
I also added the two methods to the Initialize start method of the class so they both get called upon loading the moodController

**Review by redjen — APPROVED** [review]
Approving and merging. I'll add the lines for testing.

**Pull request merged**